docs/synapse/userguides/storm_ref_lift.rstorm

```diff
@@ -1,319 +1,402 @@
 .. highlight:: none
 
 .. storm-cortex:: default
+.. storm-pre:: $propinfo = $lib.dict(doc="The VirusTotal reputation score.") $lib.model.ext.addFormProp(file:bytes, _virustotal:reputation, (int, $lib.dict()), $propinfo)
```

Comment:
This is somewhat a preference thing, `$lib.dict` works fine but we usually use JSON syntax for dictionaries now:
```suggestion
.. storm-pre:: $propinfo = ({"doc": "The VirusTotal reputation score."}) $lib.model.ext.addFormProp(file:bytes, _virustotal:reputation, (int, ({})), $propinfo)
```